share/mrdocs/addons/generator/asciidoc/partials/signature/alias.adoc.hbs

```diff
@@ -0,0 +1,5 @@
+{{!-- alias.adoc.hbs --}}
+alias {{symbol.name}} = {{>declarator symbol.aliasedType}}
```

Comment:
`AliasInfo` represents namespace aliases, so this should be `namespace`, not `alias`